builder/digitalocean/config.go

```diff
@@ -94,6 +94,9 @@ type Config struct {
 	// instead of the public IP. Before using this, private_networking should be enabled.
 	ConnectWithPrivateIP bool `mapstructure:"connect_with_private_ip" required:"false"`
 
+	// The private key algorithm to use; supports "rsa", "ecdsa", "ed25519", defaults to "rsa".
+	SSHTemporaryKeyPairAlgorithm string `mapstructure:"temporary_key_pair_algorithm" required:"false"`
```

Comment:
Other builders seem to be using `temporary_key_pair_type`, we should probably use that here too for consistency.

Reply:
Removed it since it's inherited from `communicator.Config`